pkg/edged/containers/container_manager.go

```diff
@@ -178,13 +175,13 @@ func NewContainerManager(runtimeService cri.RuntimeService, livenessManager prob
 		livenessManager:          livenessManager,
 	}
 	if devicePluginEnabled {
-		devicePluginManager, err = deviceplugin.NewManagerImpl()
+		devicePluginManager, cmErr = deviceplugin.NewManagerImpl()
```

Comment:
the device plugin code looks similar from container_manager_linux impl....is our device plugin implementation different from kubelet ?

Reply:
The implementation of kubelet is far more than what we have in edged. Hence they are different in functionality point of view. Hence it is ok here.